### DISCUSSION TITLE
after update to version 0.09 can t connect to wallet

### DISCUSSION THREAD

**Discussion by AlexeiKaDev** [Chia-Network/chia-blockchain#7479] [Support]
Hi. After update to version 0.09 can t connect to wallet
![image](https://user-images.githubusercontent.com/45790487/125564226-4a24bb49-68ff-46e8-a397-ca4340f337b6.png)

**Comment by d1Mm** [comment]
wrong repository
try to ask it [here](https://github.com/silicoin-network/silicoin-blockchain/issues)
